tests/integration/targets/activation/tasks/main.yml

```diff
@@ -404,6 +411,11 @@
         that:
           - event_stream_name in (source_mappings_dict | map(attribute='event_stream_name') | list)
 
+    - name: Assert that swap single source flag is disabled
+      ansible.builtin.assert:
+        that:
+          - not _result_activation_info.activations[0].swap_single_source
```

Comment:
We don't need this assertion, it is a feature of the collection for configuring event streams. not a field in the backend